src/app/orderConfirmationPickUp/page.tsx

```diff
@@ -66,49 +76,55 @@ export default function OrderConfirmationPickUp() {
   return (
     <div>
       <NavBar />
-      <BackButton destination="./storefront" />
-      <CenterBox>
-        <OutterBox>
-          <Heading3Bold>
-            Thank you, {user?.first_name}. Your order has been submitted.
-          </Heading3Bold>
-
-          <OutterFavoriteDiv>
-            <ColDiv>
-              {/** change this to order number! */}
-              <DateText>Order No. {user?.cart_id}</DateText>
-              {/** got the date but please clean up the date format :) */}
-
-              <Body2Bold>Pick Up: {organizePickupTime()}</Body2Bold>
-            </ColDiv>
-            {/** mess w/ the height of the scrollDiv so that the locationn stays constant :) */}
-
-            <ScrollDiv>
-              {Cart.map(cartItem => (
-                <FavoriteDiv key={cartItem.id}>
-                  <img
-                    src={cartItem.photo}
-                    alt={cartItem.name}
-                    style={{
-                      width: '150px',
-                      height: '150px',
-                      marginLeft: '80px',
-                      marginRight: '100px',
-                    }}
-                  />
-                  <LabelBox>
-                    <Label>{cartItem.name}</Label>
-                    <p>Category: {cartItem.category}</p>
-                  </LabelBox>
-                </FavoriteDiv>
-              ))}
-            </ScrollDiv>
-            <AddressDiv>
-              <Body2>Location: 3170 23rd Street, San Francisco, CA 94110</Body2>
-            </AddressDiv>
-          </OutterFavoriteDiv>
-        </OutterBox>
-      </CenterBox>
+      <CenterDiv>
+        <PageDiv>
+          <BackButtonDiv>
+            <BackButton destination="./storefront" />
+          </BackButtonDiv>
+          <BottomColumnDiv>
+            <LeftColumnDiv>
+              <TextDiv>
+                <Heading3Bold>Your order has been submitted</Heading3Bold>
+              </TextDiv>
+              <OutterFavoriteDiv>
```

Comment:
can you change the name of this div. I think in the past, someone might've copied the favorites style but we should prob rename them